server/src/main/java/org/cloudfoundry/identity/uaa/oauth/KeyInfo.java

```diff
@@ -188,7 +184,9 @@ private Optional<X509Certificate> getValidX509Certificate(String pemEncoded) {
                 x509Certificate.checkValidity();
                 return Optional.of(x509Certificate);
             }
-        } catch (RuntimeException | CertificateExpiredException | CertificateNotYetValidException e) { } // ignore
+        } catch (RuntimeException | CertificateExpiredException | CertificateNotYetValidException e) {
+            // ignore
+        }
```

Comment:
Not sure if you have seen this question. Can you answer it when you have chance?

Reply:
I removed the changes. We dont have a logger in KeyInfo and adding one for this would be too much , at least this is not important to know, so if there would be a logger in the class I could to a simply log.

So I would like to stay as it is. The meaning is, if the x509 is invalid , it wont be used for JWT keys